packages/sync/src/modules/class-callables.php

```diff
@@ -370,8 +370,13 @@ public function set_plugin_action_links() {
 			/** This filter is documented in src/wp-admin/includes/class-wp-plugins-list-table.php */
 			$action_links = apply_filters( 'plugin_action_links', $action_links, $plugin_file, null, 'all' );
 			/** This filter is documented in src/wp-admin/includes/class-wp-plugins-list-table.php */
-			$action_links           = apply_filters( "plugin_action_links_{$plugin_file}", $action_links, $plugin_file, null, 'all' );
-			$action_links           = array_filter( $action_links );
+			$action_links = apply_filters( "plugin_action_links_{$plugin_file}", $action_links, $plugin_file, null, 'all' );
+			// Verify $action_links is still an array to resolve warnings from filters not returning an array.
+			if ( is_array( $action_links ) ) {
+				$action_links = array_filter( $action_links );
+			} else {
+				$action_links = array();
```

Comment:
Seems like there should be a `_doing_it_wrong()` or something though, rather than silently ignoring the problem.

Reply:
Hey Brad, in this case we are simply hardening the code to not throw a warning if a plugin/theme is modifying plugin_action_links or plugin_action_links_{$plugin_file}. There is nothing directly to call _doing_it_wrong() on.